python/paddle/tensor/creation.py

```diff
@@ -2367,3 +2367,53 @@ def triu_indices(row, col=None, offset=0, dtype='int64'):
             attrs={'row': row, 'col': col, 'offset': offset, 'dtype': dtype},
         )
     return out
+
+
+def polar(abs, angle, name=None):
+    """Return a Cartesian coordinates corresponding to the polar coordinates compelx tensor given the ``abs`` and ``angle`` component.
+
+    Args:
+        abs (Tensor): The abs component. The data type should be 'float32' or 'float64'.
+        angle (Tensor): The anglee component. The data type should be the same as ``abs``.
+        name (str, optional): For details, please refer to :ref:`api_guide_Name`. Generally, no setting is required. Default: None.
+
+    Returns:
+        Tensor: The output tensor. The data type is 'complex64' or 'complex128', with the same precision as ``abs`` and ``angle``.
+
+    Note:
+        ``paddle.complex`` supports broadcasting. If you want know more about broadcasting, please refer to `Introduction to Tensor`_ .
+
+        .. _Introduction to Tensor: ../../guides/beginner/tensor_en.html#chapter5-broadcasting-of-tensor
+
+    Examples:
+        .. code-block:: python
+
+            import paddle
+            import numpy as np
+
+            abs = paddle.to_tensor([1, 2], dtype=paddle.float64)
+            angle = paddle.to_tensor([np.pi / 2, 5 * np.pi / 4], dtype=paddle.float64)
+            out = paddle.polar(abs, angle)
+            print(out)
+            # Tensor(shape=[2], dtype=complex128, place=Place(cpu), stop_gradient=True,
+            #       [ (6.123233995736766e-17+1j) ,
+            #       (-1.4142135623730954-1.414213562373095j)])
+    """
+    check_variable_and_dtype(abs, 'abs', ['float32', 'float64'], 'paddle.polar')
+    check_variable_and_dtype(
+        angle, 'angle', ['float32', 'float64'], 'paddle.polar'
+    )
+
+    if abs.dim() != angle.dim():
+        raise ValueError(
+            f"abs and angle tensor must be the same dimension, but got abs dim {abs.dim()} and angle dim {angle.dim()}."
+        )
+
+    if abs.shape != angle.shape:
+        raise ValueError(
+            f"abs and angle tensor must be the same shape, but got abs shape {abs.shape} and angle dim {angle.shape}."
+        )
+
+    real = paddle.multiply(abs, paddle.cos(angle))
```

Comment:
it is possible to use just one multiplication

paddle.multiply(abs, paddle.complex(paddle.cos(angle), paddle.cos(angle))

Reply:
okay, it looks better actually. fixed